File: SkaiLab.Invoice.Models/Plan.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SkaiLab.Invoice.Models
{
    public class Plan
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public double MonthlyPrice { get; set; }
        public double MonthlyRenewalPrice { get; set; }
        public double YearlyPrice { get; set; }
        public double YearlyRenewalPrice { get; set; }
    }
}

File: SkaiLab.Invoice/ClientApp/src/app/models/plan.ts
export class Plan{
    id:number;
    name:string;
    monthlyPrice:number;
    monthlyRenewalPrice:number;
    yearlyPrice:number;
    yearlyRenewalPrice:number;
    yearlySavePercent:number;
    constructor(){
        this.id=0;
        this.name="";
        this.monthlyPrice=0;
        this.monthlyRenewalPrice=0;
        this.yearlyPrice=0;
        this.yearlyRenewalPrice=0;
        this.yearlySavePercent=0;
    }
}